pvlib/tracking.py

```diff
@@ -570,3 +522,151 @@ def singleaxis(apparent_zenith, apparent_azimuth,
         out = {k: np.where(zen_gt_90, np.nan, v) for k, v in out.items()}
 
     return out
+
+
+def calc_axis_tilt(slope_azimuth, slope_tilt, axis_azimuth):
+    """
+    Calculate tracker axis tilt in the global reference frame when on a sloped
+    plane.
+
+    Parameters
+    ----------
+    slope_azimuth : float
+        direction of normal to slope on horizontal [degrees]
+    slope_tilt : float
+        tilt of normal to slope relative to vertical [degrees]
+    axis_azimuth : float
+        direction of tracker axes on horizontal [degrees]
+
+    Returns
+    -------
+    axis_tilt : float
+        tilt of tracker [degrees]
+
+    See also
+    --------
+    pvlib.tracking.singleaxis
+    pvlib.tracking.calc_cross_axis_tilt
+
+    Notes
+    -----
+    See [1]_ for derivation of equations.
+
+    References
+    ----------
+    .. [1] Kevin Anderson and Mark Mikofski, "Slope-Aware Backtracking for
+       Single-Axis Trackers", Technical Report NREL/TP-5K00-76626, July 2020.
+       https://www.nrel.gov/docs/fy20osti/76626.pdf
+    """
+    delta_gamma = axis_azimuth - slope_azimuth
+    # equations 18-19
+    tan_axis_tilt = cosd(delta_gamma) * tand(slope_tilt)
+    return np.degrees(np.arctan(tan_axis_tilt))
+
+
+def _calc_tracker_norm(ba, bg, dg):
+    """
+    Calculate tracker normal, v, cross product of tracker axis and unit normal,
+    N, to the system slope plane.
+
+    Parameters
+    ----------
+    ba : float
+        axis tilt [degrees]
+    bg : float
+        ground tilt [degrees]
+    dg : float
+        delta gamma, difference between axis and ground azimuths [degrees]
+
+    Returns
+    -------
+    vector : tuple
+        vx, vy, vz
+    """
+    cos_ba = cosd(ba)
+    cos_bg = cosd(bg)
+    sin_bg = sind(bg)
+    sin_dg = sind(dg)
+    vx = sin_dg * cos_ba * cos_bg
+    vy = sind(ba)*sin_bg + cosd(dg)*cos_ba*cos_bg
+    vz = -sin_dg*sin_bg*cos_ba
+    return vx, vy, vz
+
+
+def _calc_beta_c(v, dg, ba):
+    """
+    Calculate the cross-axis tilt angle.
+
+    Parameters
+    ----------
+    v : tuple
+        tracker normal
+    dg : float
+        delta gamma, difference between axis and ground azimuths [degrees]
+    ba : float
+        axis tilt [degrees]
+
+    Returns
+    -------
+    beta_c : float
+        cross-axis tilt angle [radians]
+    """
+    vnorm = np.sqrt(np.dot(v, v))
+    beta_c = np.arcsin(
+        ((v[0]*cosd(dg) - v[1]*sind(dg)) * sind(ba) + v[2]*cosd(ba)) / vnorm)
+    return beta_c
+
+
+def calc_cross_axis_tilt(
+        slope_azimuth, slope_tilt, axis_azimuth, axis_tilt):
+    """
+    Calculate the angle, relative to horizontal, of the line formed by the
+    intersection between the slope containing the tracker axes and a plane
```

Comment:
```suggestion
    intersection between the plane containing the tracker axes and a plane
```

Reply:
I opted leave the word "slope" because I thought "plane ... plane" was a tongue twister and IMHO didn't add enough value to make the change worthwhile. Feel free to overrule me. 😉